autogpt_platform/backend/backend/data/model.py

```diff
@@ -260,15 +260,32 @@ class OAuthState(BaseModel):
 
 class UserMetadata(BaseModel):
     integration_credentials: list[Credentials] = Field(default_factory=list)
+    """⚠️ Deprecated; use `UserIntegrations.credentials` instead"""
     integration_oauth_states: list[OAuthState] = Field(default_factory=list)
+    """⚠️ Deprecated; use `UserIntegrations.oauth_states` instead"""
```

Comment:
can we get the proper deprecated wrapper so the type checker and ide tags these

Reply:
only works on functions and methods afaik